### PR TITLE
Update gRPC docs for preview 7

### PR DESCRIPTION
**Pull request by JamesNK**
* Update gRPC tutorial client to use HTTPS
* Update articles to refer to metapackage
* Fix https://github.com/aspnet/AspNetCore.Docs/issues/13037
* Fix https://github.com/aspnet/AspNetCore.Docs/issues/12697 (sample now uses HttpClient with HTTPS)
* Fix https://github.com/grpc/grpc-dotnet/issues/358

# Do not merge until preview 7 is released.

**Internal review URLs**:
* [gRPC services with ASP.NET Core](https://review.docs.microsoft.com/en-us/aspnet/core/grpc/aspnetcore?view=aspnetcore-3.0&branch=pr-en-us-13051&tabs=visual-studio)
* [gRPC services with C#](https://review.docs.microsoft.com/en-us/aspnet/core/grpc/basics?view=aspnetcore-3.0&branch=pr-en-us-13051)
* [Tutorial: Create a gRPC client and server](https://review.docs.microsoft.com/en-us/aspnet/core/tutorials/grpc/grpc-start?view=aspnetcore-3.0&branch=pr-en-us-13051&tabs=visual-studio)

### CODE REVIEW

**Comment by JamesNK** [comment]
@Rick-Anderson There is a new validation warning related to tabs. I don't know what the fix is. Could you resolve it?

**Comment by Rick-Anderson** [comment]
@JamesNK  it's a false positive. I'm not sure why it's only showing now. The tabs are fine, you can ignore that. Did you see my commit that triggered the new build?

**Comment by scottaddie** [comment]
@JamesNK Can you please use the *release/3.0-preview7* branch for this PR? I plan to merge that branch into *master* when Preview 7 ships.

**Comment by JamesNK** [comment]
Rebased.

Also updated with latest packages and template changes.

File: aspnetcore/grpc/aspnetcore/sample/GrcpService/GreeterService2.cs [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Greet;
+using Grpc.Core;
+
+namespace GrpcGreeter
+{
+    #region snippet
+    public class GreeterService : Greeter.GreeterBase
```
Comment (shirhatti) [inline aspnetcore/grpc/aspnetcore/sample/GrcpService/GreeterService2.cs]: Is it still `Greeter.GreeterBase`? I thought we changed the C# namespace to match your project now

File: aspnetcore/tutorials/grpc/grpc-start/sample/GrpcGreeterClient/GrpcGreeterClient.csproj [hunk 2]
```diff
@@ -7,11 +7,8 @@
 
   <ItemGroup>
     <PackageReference Include="Google.Protobuf" Version="3.8.0" />
-    <PackageReference Include="Grpc.Net.Client" Version="0.1.21-pre1" />
-    <PackageReference Include="Grpc.Tools" Version="1.21.0">
-      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
-      <PrivateAssets>all</PrivateAssets>
-    </PackageReference>
+    <PackageReference Include="Grpc.Net.Client" Version="0.1.22-pre2" />
+    <PackageReference Include="Grpc.Tools" Version="1.22.0-pre1" PrivateAssets="All" />
```
Comment (JunTaoLuo) [inline aspnetcore/tutorials/grpc/grpc-start/sample/GrpcGreeterClient/GrpcGreeterClient.csproj]: 1.22.0?

File: aspnetcore/grpc/aspnetcore/sample/GrcpService/GreeterService2.cs (hunk 1 above)
Comment (JunTaoLuo) [reply aspnetcore/grpc/aspnetcore/sample/GrcpService/GreeterService2.cs]: GreeterBase is a nested class of Greeter. The Greeter's namespace has been updated to the match the project.

Comment (shirhatti) [reply aspnetcore/grpc/aspnetcore/sample/GrcpService/GreeterService2.cs]: Whoops 🤦‍♂

**Comment by JamesNK** [comment]
@scottaddie Can be merged into the preview 7 branch.

**Comment by scottaddie** [comment]
Thank you, @JamesNK!

**Pull request merged**